10up/insecure-content-warning
# Update import statement for PluginPostStatusInfo

### Description of the Change

Update the import statement for `PluginPostStatusInfo` component to use unified extensibility API.


Closes #173

### How to test the Change

No functional changes to the plugin. 
Adding an image with HTTP source should trigger the notice in the sidebar when trying to save a post.

### Changelog Entry

> Changed - Changed import of `PluginPostStatusInfo` component from `@wordpress/edit-post` to `@wordpress/editor`


### Credits

Props @username, ...


### Checklist:
